backend/src/test/java/gov/cdc/usds/simplereport/service/sms/SmsServiceTest.java

```diff
@@ -150,4 +166,53 @@ void sendPatientLinkSmsUpdatesPatientLinkExpiry() throws NumberParseException {
     // THEN
     assertTrue(previousExpiry.before(updatedPatientLink.getExpiresAt()));
   }
+
+  @Test
+  @WithSimpleReportStandardAllFacilitiesUser
+  void sendPatientLinkSms_returnsSmsDeliveryResults() throws NumberParseException {
+    // GIVEN
+    _person = _dataFactory.createFullPerson(_org);
+    createTestOrderAndPatientLink(_person);
+
+    // WHEN
+    when(mockTwilio.send(any(), fromNumber.capture(), message.capture()))
+        .thenReturn("some-twilio-id-10");
+
+    var sut = _smsService.sendToPatientLink(_patientLink.getInternalId(), "it's a test!");
+
+    // THEN
+    assertEquals(1, sut.size());
+    var result = sut.get(0);
+    assertEquals(_person.getPrimaryPhone().getNumber(), result.getTelephone());
+    assertEquals("some-twilio-id-10", result.getMessageId());
+    assertEquals(true, result.getDeliverySuccess());
+  }
+
+  @Test
+  @WithSimpleReportStandardAllFacilitiesUser
```

Comment:
**suggestion:** consider adding both land and mobile numbers and confirming text messages are only attempted to be sent to mobile numbers

Reply:
Excellent call 🦖 

Resolved by a8afe79.